comic_identity_engine/jobs/tasks.py

```
@@ -0,0 +1,857 @@
+                        "cover_date": (
+                            issue.cover_date.isoformat() if issue.cover_date else None
+                        ),
+                        "upc": issue.upc,
+                        "series_id": str(issue.series_run_id),
+                        "series_title": series.title if series else None,
+                        "series_start_year": series.start_year if series else None,
+                        "publisher": series.publisher if series else None,
+                    }
+                    records.append(record)
+
+            export_dir = Path("/tmp/comic_exports")
```

Ruff (0.15.2)

[error] Probable insecure usage of temporary file or directory: "/tmp/comic_exports"

(S108)